build-config/resources/Ballerina.toml

```diff
@@ -5,12 +5,12 @@ name = "hubspot.crm.engagement.notes"
 version = "@toml.version@"
 license = ["Apache-2.0"]
 authors = ["Ballerina"]
-keywords = [] # TODO: Add keywords
+keywords = ["hubspot", "crm", "engagement", "notes"]
 # icon = "icon.png" # TODO: Add icon
```

Comment:
I think you have missed to add an `icon.png` file to the `ballerina` directory. Please add it and uncomment this line.